tests/suites/hup/suite.js

```diff
@@ -20,6 +20,50 @@ const pipeline = require('bluebird').promisify(stream.pipeline);
 // required to use skopeo for loading the image
 const exec = require('bluebird').promisify(require('child_process').exec);
 
+// copied from the SV
+// https://github.com/balena-os/balena-supervisor/blob/master/src/config/backends/config-txt.ts
+// TODO: retrieve this from the API
+const supportsBootConfig = (deviceType) => {
```

Comment:
@klutchell I like this more, it's apparent that it's a device specific customization that's only applied in certain cases. I also like that we left a comment describing the better/preferred way to do this when we find the time.